src/schema/v2/sale/index.ts

```diff
@@ -234,6 +237,62 @@ export const SaleType = new GraphQLObjectType<any, ResolverContext>({
           }
         },
       },
+      auctionsDetailFormattedStartDateTime: {
```

Comment:
It's a bummer that we couldn't reuse the helpers we already have. I think fine for now but we should see if there's any consolidation of logic we can follow up with because now there's three methods (`cascadingFormattedStartDateTime`, `formattedStartDateTime` and `auctionsDetailFormattedStartDateTime`) that all do pretty similar logic to show states based on whether a lot is not yet open, open but lots are all still open, lots are currently closing, or sale is totally over. I see how some of the formatting is different so def wasn't an easy way around this.

Reply:
yeah I mean essentially the copy was different and I tried to extract it out, because there is over lap, based on the context of whether we were talking about looking at the sale index vs the sale details, but it just ended up feeling like I was playing JS golf and just over engineering it, so I split it out to a separate method